content/en/docs/setup/install/operator/test.sh

```diff
@@ -22,9 +22,11 @@ set -e
 set -u
 set -o pipefail
 
-fullVersion=$(type snip_canary_upgrade_init | sed '1,3d;$d' | sed 's/.*istio-\(.*\)\/bin.*/\1/') # 1.18.0
+source "content/en/boilerplates/snips/args.sh"
+
+fullVersion="${bpsnip_args_istio_full_version}"
 fullVersionRevision=$(echo "$fullVersion" | sed -r 's/[.]+/-/g' ) # 1-18-0
```

Comment:
Why not a simpler (more obvious) sed command?
```suggestion
fullVersionRevision=$(echo "$fullVersion" | sed 's/\./-/g' ) # 1-18-0
```

Reply:
addressed in ce5401bc7428932de63cef0afe3adc24ace1f1e7